### PR TITLE
Move /cloud/capabilities and /cloud/user to Core

### PR DESCRIPTION
**Pull request by rullzer**
Requirese https://github.com/nextcloud/server/pull/691

My ongoing quest.

Basically move `/cloud/capabilities` and `/cloud/user` to the Core app. Which means we can use the AppFramework and get rid of deprecated API calls.

Lets see what CI says here.

### CODE REVIEW

File: core/Controller/OCSController.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	public function getCurrentUser() {
+		$userObject = $this->userSession->getUser();
+		$data  = [
+			'id' => $userObject->getUID(),
+			'display-name' => $userObject->getDisplayName(),
+			'email' => $userObject->getEMailAddress(),
+		];
+		return new DataResponse(['data' => $data]);
+	}
+}
```
Comment (LukasReschke) [inline core/Controller/OCSController.php]: new line?

Comment (rullzer) [inline core/Controller/OCSController.php]: fixed

**Comment by rullzer** [comment]
#691 is in.
Time for review!

CC: @MorrisJobke @LukasReschke @BernhardPosselt @icewind1991 @nickvergessen

**Comment by LukasReschke** [comment]
👍

**Comment by MorrisJobke** [comment]
Tested and the API works as before 👍

**Pull request merged**